test/object_store/run

```diff
@@ -62,6 +63,40 @@ def check_with_cql(ip, ssl):
     return checker
 
 
+def run_with_dir(run_cmd_gen, run_dir):
+    print(f'Start scylla (dir={run_dir}')
+    mask = signal.pthread_sigmask(signal.SIG_BLOCK, {})
+    signal.pthread_sigmask(signal.SIG_BLOCK, {signal.SIGINT, signal.SIGQUIT, signal.SIGTERM})
+    pid = os.fork()
```

Comment:
The original cql-pytest's flushed stdios here. This helper doesn't. Is cql-pytest's code redundant or is this place missing some important fix?

Reply:
flushing stdio before a fork is important, otherwise both parent and child will inherit the same stdio buffer and both will flush it resulting in the same message printed twice.
To see a difference, please redirect the output to a file or pipe, otherwise the output is always line-buffered so you'll not notice the problem.

Reply:
the former. i don't understand why it's necessary to flush the stderr and stdout before calling `os.fork()`.

Reply:
@tchaikov please read my explanation

Reply:
okay. makes sense to me. thank you Nadav. i am keeping the `flush()` statements.